src/riak_kv_entropy_manager.erl

```diff
@@ -189,6 +208,15 @@ expire_trees() ->
 clear_trees() ->
     gen_server:cast(?MODULE, clear_trees).
 
+get_version() ->
```

Comment:
Would be nice to have type specs on these as long as we're adding new API functions. I wouldn't let it stop this from passing review since none of the other API calls in this module seem to have type specs, but just thought I'd suggest it in case you're waiting for more review comments and you have a chance to add them :)